tests/dotnet/UnitTests/Extensions.cs

```diff
@@ -1,4 +1,5 @@
 using System;
+using System.Diagnostics;
```

Comment:
There are no other changes in this file, so I don't think you need to add a `using`.